qoqo/tests/integration/operations/bosonic_operations.rs

```diff
@@ -391,8 +389,12 @@ fn test_pyo3_mode(mode: usize, input_operation: Operation) {
     pyo3::prepare_freethreaded_python();
     Python::with_gil(|py| {
         let operation = convert_operation_to_pyobject(input_operation).unwrap();
-        let mode_op: usize =
-            usize::extract_bound(&operation.call_method0(py, "mode").unwrap().bind(py)).unwrap();
+        let mode_op: usize = operation
+            .call_method0(py, "mode")
+            .unwrap()
+            .bind(py)
+            .extract()
+            .unwrap();
```

Comment:
Why is this formulation better?

Reply:
I think both are fine but this one can be more readable